_includes/card.html

```diff
@@ -8,21 +8,22 @@
   {% assign wds_grid_class = 'usa-width-one-third' %}
 {% endif %}
 
-{% assign project_type = include.project_type | default: 'none' %}
-
-<article class="card {{ wds_grid_class }}" data-bucket="{{ project_type }}">
-  <a class="card-link" href="{{ card_link }}">
-    <div role="img" title="{{ include.image_alt }}" class="card-image-bg" style="background-image: url({{ site.baseurl }}{{ include.image_src }});">
-    </div>
+<article class="card {{ wds_grid_class }}">
+  <div class="card-link" onclick='window.location = "{{ include.link }}";' tabindex="-1">
+    <div role="img"
+         title="{{ include.image_alt }}"
```

Comment:
So it seems like you set your alt image tags manually in the templates, but I just want to mention that you shouldn't be using "Image of..." bc that's redundant as it being an image is already announced to users with the `role="img"` property. More info here: http://webaim.org/techniques/alttext/